### PR TITLE
Something in docs may cause confusion

### PR DESCRIPTION
**Pull request by Dios314**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

在文档基本样例_ 延时消息队列中
可能是因为疏忽 未添加consumer/producer.setNamesrvAddr("localhost:9876"); 这可能会导致初学者产生困惑
并最终导致验证失败
In Example_Delay
maybe because of carelessness,
there is no code to specify name server addresses.
//producer.setNamesrvAddr("localhost:9876");  
//consumersetNamesrvAddr("localhost:9876"); 
This may cause confusion and lead to failure to verification. 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: docs/cn/RocketMQ_Example.md [hunk 1]
```diff
@@ -440,6 +440,8 @@ public class ScheduledMessageConsumer {
    public static void main(String[] args) throws Exception {
       // 实例化消费者
       DefaultMQPushConsumer consumer = new DefaultMQPushConsumer("ExampleConsumer");
+      // 设置NameServer的地址
+      consumer.setNamesrvAddr("localhost:9876");
```
Comment (hzh0425) [inline docs/cn/RocketMQ_Example.md:444]: It is better to annotate in English

File: docs/cn/RocketMQ_Example.md [hunk 2]
```diff
@@ -471,6 +473,8 @@ public class ScheduledMessageProducer {
    public static void main(String[] args) throws Exception {
       // 实例化一个生产者来产生延时消息
       DefaultMQProducer producer = new DefaultMQProducer("ExampleProducerGroup");
+      // 设置NameServer的地址
+      producer.setNamesrvAddr("localhost:9876");
```
Comment (hzh0425) [inline docs/cn/RocketMQ_Example.md:477]: ditto

File: docs/cn/RocketMQ_Example.md (hunk 1 above)
Comment (li-xiao-shuang) [reply docs/cn/RocketMQ_Example.md:444]: > It is better to annotate in English

agree

Comment (Dios314) [reply docs/cn/RocketMQ_Example.md:444]: but it is a documention wrote in Chinese.

**Pull request merged**